bundles/binding/org.openhab.binding.velux/ESH-INF/i18n/velux_de.properties

```diff
@@ -0,0 +1,66 @@
+
+# binding
+binding.velux.name = Velux Binding  (1.x)
+binding.velux.description = Ein openHAB1 Binding f�r das Velux KLF200 Kopplungselement. Das Velux Binding kommuniziert �ber ein Velux Kopplungselement mit den unterschiedlichen Velux-Ger�ten zur Steuerung von Fenstern, Roll�den und anderen Ger�ten. Beispielsweise kann ein KLF 200 als Schnittstelle zwischen Hausleitsystemen und VELUX INTEGRA Produkten mit Funkanbindung basierend auf dem io-homecontrol-Standard eingesetzt werden.
+
+# thing types
```

Comment:
Could you please also remove all thing and channel type translations here as well?

Reply:
Removed.